pkg/routeagent/controllers/route/route_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package routecontroller_test
```

Comment:
For this to work you need to change this to "route"

Otherwise if you do make test (you may need to rebase on origin/master for that to work, you get...) 
```
Running tests in . ./pkg/...
Failed to compile route:

can't load package: package github.com/rancher/submariner/pkg/routeagent/controllers/route: found packages route (route.go) and routecontroller (route_suite_test.go) in /go/src/github.com/rancher/submariner/pkg/routeagent/controllers/route
```

after making that change: 

````